README.md

```diff
@@ -7,7 +7,7 @@ the below**
 
 [You can find the engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
 
-2. The card wall is here: <please update>
+2. The card wall is here: https://trello.com/b/qGXi1GtK/acebook
```

Comment:
This is linked to in it's own section at the bottom of the readme. To avoid repetition, can we remove this one?

Reply:
We talked with Lima that the link should be here and the one in the bottom should be deleted

Reply:
Great - thanks, I've updated that.